bears/java/JavaPMDBear.py

```diff
@@ -17,22 +17,14 @@ class JavaPMDBear:
     """
 
     LANGUAGES = {'Java'}
+    REQUIREMENTS = {MavenRequirement('net.sourceforge.pmd:pmd', '5.6.0'),
```

Comment:
this would be a breaking change.
`MavenRequirement` needs to be an optional way to obtain `pmd`.

And I am pretty sure that the maven `pmd` doesnt install a `pmd` executable.

i.e. you need to test that old and new work, which is why this is listed as medium difficulty